builder/ar.go

```diff
@@ -12,6 +12,7 @@ import (
 	"path/filepath"
 	"time"
 
+	wasm "github.com/aykevl/go-wasm"
```

Comment:
Note that the package is already named "wasm", so this would be fine:

```suggestion
	"github.com/aykevl/go-wasm"
```

(Many people think the directory name is used, but that's not true: the name from `package wasm` is used).